GitHubRepositories.ps1

```diff
@@ -175,7 +189,10 @@ function New-GitHubRepository
         'NoStatus' = (Resolve-ParameterWithDefaultConfigurationValue -BoundParameters $PSBoundParameters -Name NoStatus -ConfigValueName DefaultNoStatus)
     }
 
-    return Invoke-GHRestMethod @params
+    $result =  Invoke-GHRestMethod @params
+    $result.PSObject.TypeNames.Insert(0,$script:GitHubRepositoryTypeName)
```

Comment:
nit: Here and elsewhere, can you add a space after the comma

```suggestion
    $result.PSObject.TypeNames.Insert(0, $script:GitHubRepositoryTypeName)
```

Reply:
Done